src/fee.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+import domains
+
+
+class Fee:
+    def __init__(self, sell_token, buy_token, sell_amount, kind, buy_amount):
+        self.sell_token = sell_token
+        self.buy_token = buy_token
+        self.sell_amount = sell_amount
+        self.buy_amount = buy_amount
+        self.kind = kind
+        self.base_url = domains.base_url
+
+    def get_fee(self) -> "uint256":
+        url = "{}fee?sellToken={}&buyToken={}&amount={}&kind={}".format(
+            self.base_url,
+            self.sell_token,
+            self.buy_token,
+            self.buy_amount,
+            self.kind
+        )
+
+        request_fee = requests.get(url)
+        return int(request_fee.json().get("amount", 0))
```

Comment:
Rather than an in statement to handle the case when the response json doesn't contain the "amount" key, we take advantage of python's native `.get` method that works on both type dict and whatever it interprets json as. 

get is used to fetch the value at the key specified or use the default value provided as the second parameter. In our case we were using 0. 

Also, I don't think that the orderbook api will accept a fee of zero so this mean we are just delaying the inevitable failure of this script by falling back on a default value we know will not work.